Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM python:3.6.3-alpine3.6
+FROM python:3.6.3
 
 # Git is required for repo2docker to work
-RUN apk add --no-cache git
+#RUN apk add --no-cache git
```

Comment:
any reason that we wouldn't remove this line (and the one above) vs. commenting it out. Are we going to reenable after the merge?

Reply:
@willingc good catch, I've removed those!